resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -232,7 +232,7 @@ function task_termini {
     #
     # This also allows us to prefer PE ruby over system ruby, so this will work
     # with PE as well
-    local rubylibdir=${rubylibdir:=$(/opt/puppetlabs/puppet/bin/ruby -rrbconfig -e "puts RbConfig::CONFIG['vendordir']")}
+    local rubylibdir=${rubylibdir:=/opt/puppetlabs/puppet/lib/ruby/vendor_ruby}
```

Comment:
Minor: this has the expected behavior but is redundant

```${rubylibdir:=/opt/puppetlabs/puppet/lib/ruby/vendor_ruby}```

will set the global ```rubylibdir``` variable by itself, so

```local rubylibdir=${rubylibdir:=/opt/puppetlabs/puppet/lib/ruby/vendor_ruby}```

will set the *local* rubylibdir to the value of the global rubylibdir

I suggest:
```local rubylibdir=${rubylibdir:-/opt/puppetlabs/puppet/lib/ruby/vendor_ruby}```

However, note that this only sets the local rubylibdir, so once the function is done, the global rubylibdir goes back into scope.